packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -93,6 +99,17 @@ export class Oracle {
   async listOracles (): Promise<string[]> {
     return await this.client.call('listoracles', [], 'number')
   }
+
+  /**
+   * Returns latest raw price updates from oracles.
+   *
+   * @param {OraclePriceFeed} [priceFeed]
+   * @return {Promise<OracleRawPrice[]>}
+   */
+  async listLatestRawPrices (priceFeed?: OraclePriceFeed): Promise<OracleRawPrice[]> {
+    const params = priceFeed !== undefined ? [priceFeed] : []
```

Comment:
better to use priceFeed != null to cover null and undefined

Reply:
Ok

Reply:
@thedoublejay hmm.. actually `null` doesn't undefined, they are diff 
the best way is actually checking the condition by `!priceFeed` which covers price != undefined && price != null && price != ""
but ts-standard will auto convert `!priceFeed` to `priceFeed !== undefined`
i'm fine with !== null as well
will making another advance ts-lint and formatting to standardized it

Reply:
@canonbrother `null` covers `undefined` as long as you don't use strict equality. 
![image](https://user-images.githubusercontent.com/25013382/121637436-f00cd500-cabb-11eb-87ed-bffc30e33d16.png)

Reply:
> @canonbrother `null` covers `undefined` as long as you don't use strict equality.
> ![image](https://user-images.githubusercontent.com/25013382/121637436-f00cd500-cabb-11eb-87ed-bffc30e33d16.png)
> ![image](https://user-images.githubusercontent.com/25013382/121637502-09158600-cabc-11eb-871d-79c5c2cd61ab.png)

I hate this about JavaScript, everything should be strict by default. This allows mistakes to happen because you didn't check whether you used strict or not e.g. `!=` vs `!==`, everyone assumes it's `!==` ultimately to correct misconception. 

I prefer explicit over implicit if you can use `!==` or `===` always use that. I rather see it cleaner than having to guess it. Better to code defensive than openly.

https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#explicit-over-implicit

/cc @jingyi2811 @siradji @aikchun @monstrobishi @canonbrother @izzycsy @thedoublejay @surangap @ivan-zynesis

Reply:
> @canonbrother `null` covers `undefined` as long as you don't use strict equality.
> ![image](https://user-images.githubusercontent.com/25013382/121637436-f00cd500-cabb-11eb-87ed-bffc30e33d16.png)
> ![image](https://user-images.githubusercontent.com/25013382/121637502-09158600-cabc-11eb-871d-79c5c2cd61ab.png)

Generally speaking, that's not true. For eg, just one row declaration
```
let foo: Foo | null
```
Broken JS may cover `foo != null` but
```
switch (foo) should be covered for all 3 cases
```